practice-two/src/index.html

```diff
@@ -18,7 +18,7 @@
     <header class="main-header">
       <div class="container">
         <nav class="navbar">
-          <h1 class="navbar__logo">H.</h1>
+          <a href="javascript:void(0)" class="navbar__logo">H.</a>
```

Comment:
Should have an h1 on your page. You can use `<h1><a href="javascript:void(0)" class="navbar__logo">H.</a></h1>`